invoices_app/tests.py

```diff
@@ -29,31 +38,53 @@
 )
 
 
-class TestInvoice(TestCase):
+class TestBase(TestCase):
+    def invoice_creation(self, taxpayer=None, user=None):
+        if not taxpayer:
+            taxpayer = self.taxpayer
+
+        if not user:
+            user = self.user
+
+        return {
+            'invoice_date': datetime(2007, 12, 5, 0, 0, 0, 0, UTC),
+            'invoice_type': 'A',
+            'invoice_number': '1234',
+            'po_number': '98876',
+            'currency': 'ARS',
+            'net_amount': '4000',
+            'vat': '1200',
+            'total_amount': '5200',
+            'invoice_file': 'test.pdf',
+            'taxpayer': taxpayer,
+            'user': user,
+        }
+
     def setUp(self):
```

Comment:
You should always keep `setUp` method as the first method in the class. Please move it up.